create_tarball.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+
+set -e
+
```

Comment:
Do we want to do something like:

for pilot_version in $( ls -rt1d ${cvmfs_repo}  ); do
     for cpu_arch_subdir = $( ls -rt1d ${cvmfs_repo}/${pilot_version} ); do
         make_tarball( $pilot_version, $cpu_arch_dir )
    done
done

Reply:
No, that would create a huge tarball for every CPU subdir, we only want to tar up the new stuff, to avoid having to deal with huge tarballs.

Reply:
I was thinking of implementing `make_tarball` the same way as the original code, using data from upper, but essentially building tarballs for every arch subdir for every pilot version the script finds. Also, we can now add a way to upload data to the staging bucket. I'll write some test code. :)

Reply:
https://github.com/EESSI/infrastructure/issues/25